dkg/paranet.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, manager: DefaultRequestManager):
     _register_paranet = Method(BlockchainRequest.register_paranet)
 
     def create(
-        self, ual: UAL, name: str, description: str
+        self, ual: UAL, name: str, description: str, nodes_access_policy: int, miners_access_policy: int
```

Comment:
Let's make `nodes_access_policy` and `miners_access_policy` Enums, take a look how it's done for other stuff in the `dataclasses.py`